frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ProjectActionsForm.tsx

```diff
@@ -138,6 +219,98 @@ export const ProjectActionsForm = ({
                 onBlur={(e) => handleOnBlur(() => validateName(e.target.value))}
                 autoComplete='off'
             />
+
+            <Step name='Trigger'>
+                <StyledInputDescription>
+                    Create incoming webhooks from&nbsp;
+                    <RouterLink to='/integrations/incoming-webhooks'>
+                        integrations section
+                    </RouterLink>
+                    .
+                </StyledInputDescription>
+                <GeneralSelect
+                    label='Incoming webhook'
+                    name='incoming-webhook'
+                    options={incomingWebhookOptions}
+                    value={`${sourceId}`}
+                    onChange={(v) => {
+                        setSourceId(parseInt(v));
+                    }}
+                />
+            </Step>
+
+            <Step name='When this'>
+                {filters.map((filter, index) => (
+                    <Filter
+                        index={index}
```

Comment:
Usually we specify the key at this level, not inside the component.
```suggestion
                        key={filter.id}
                        index={index}
```

Reply:
[10277ae](https://github.com/Unleash/unleash/pull/6115/commits/10277ae4c8409b21d6eb2a024b7e7abef6bf8faf)